docs/CSS.md

```diff
@@ -0,0 +1,98 @@
+### 1. `flex`
+* *elements:* none
+* *inherited:* no 
+
+#### Values
+
+* `<flex-grow>` -- defines the flex-grow of the flex item
+* `<flex-shrink>` -- defines the flex-shrink of the flex item
+* `<flex-basis>` -- defines the flex-basis of the flex item
+* `none` -- this keyword computes to 0 0 auto;
+
+### 2. `margin`
+* *elements:* all elements except elements with table display types other than table-caption, table and inline-table.
+* *inherited:* no
+
+#### Values
+
+* `<length>` -- specifies a fixed width
+* `<percentage>` -- relative to the width containing the block
+*  `auto` -- auto is replaced by some suitable value 
+
+### 3. `text-align`
+* *elements:* block-containers
+* *inherited:* yes
+
+#### Values
+* `start` -- same as left if direction is left-to-right
+* `end` -- the same as right if direction is left-to-right
```

Comment:
Know what that means? That `direction` thingy is another CSS property... You should totally look it up!